docs/src/pages/components/view/react-auto-prop-table-view.mdx

```diff
@@ -0,0 +1,48 @@
+import { Expander, ExpanderItem } from '@aws-amplify/ui-react';
+import ReactMarkdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+
+## Props
+
+<Expander type="multiple" defaultValue={['Props']}>
+  
+<ExpanderItem title="Amplify UI Props" value="Props">
+    <ReactMarkdown
+      remarkPlugins={[remarkGfm]}
+      children={`
+| Name | Type | Default | Description |
+| ---- | ---- | ------- | ----------- |
+| ref | Ref&#60;SVGSymbolElement &#124; SVGElement &#124; HTMLObjectElement &#124; HTMLElement &#124; HTMLAnchorElement &#124; ... 112 more ... &#124; SVGViewElement&#62; | / |  |
+
+      `}
+    />
+
+  </ExpanderItem>
+
+<ExpanderItem title="Layout" value="Layout">
+    <ReactMarkdown
+      remarkPlugins={[remarkGfm]}
+      children={`
+| Name | Type | Default | Description |
+| ---- | ---- | ------- | ----------- |
+| / | / | / | / |
+
+      `}
+    />
+
+  </ExpanderItem>
+
+<ExpanderItem title="Styling" value="Styling">
+    <ReactMarkdown
+      remarkPlugins={[remarkGfm]}
+      children={`
+| Name | Type | Default | Description |
+| ---- | ---- | ------- | ----------- |
+| / | / | / | / |
```

Comment:
These shouldn't be rendering if they are blank. But also View should have the full set of Styling props.

Reply:
Done